src/component/RecipeFormModal.tsx

```diff
@@ -7,7 +7,36 @@ import Modal from "./Modal";
 import { makeRecipe } from "../utils/makeRecipe";
 import ErrorMessage from "./ErrorMessage";
 
-const initialFormValue = {
+export interface RecipeFormState {
```

Comment:
With all the new types, maybe you should start moving all type definitions to another file? 🤔